### PR TITLE
Update requirements and README

### LINKED ISSUE #11: Create a requirements.txt of all libraries and versions needed to run the assistant.

### PR DESCRIPTION
**Pull request by sabderemane**
Fixes #11 
- Update requirements.txt
- Update README.md with the install

### CODE REVIEW

**Review by luminousbeam — COMMENTED** [review]
I would add here that you also need to have Pip installed and some virtual environment running for installation.

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-
+pyscreenshot==3.0
```
Comment (luminousbeam) [inline requirements.txt:1]: May I ask which operating system both you @sabderemane and @Veershah26 you are running? Even after doing an install of those three libraries, when I run main.py the assistant errors out and I see this:
```
    raise AttributeError("Could not find PyAudio; check installation")
AttributeError: Could not find PyAudio; check installation
```
When I try to manually do a `pip install pyaudio` I get another error and I cannot procede. I am using a MacOS w/Intel processor.
```
Collecting PyAudio
  Using cached PyAudio-0.2.11.tar.gz (37 kB)
Building wheels for collected packages: PyAudio
  Building wheel for PyAudio (setup.py) ... error
  ERROR: Command errored out with exit status 1:...
```

Comment (sabderemane) [reply requirements.txt:1]: Which python version are you using ? I'm on same system (Big Sur)

Comment (luminousbeam) [reply requirements.txt:1]: Also BigSur 11.6. I just found a fix looking at the pyAudio instructions here https://people.csail.mit.edu/hubert/pyaudio/
```
brew install portaudio
pip install pyaudio
```

Comment (sabderemane) [reply requirements.txt:1]: Nope i didn't have to when I've done the installation. I have to check which are necessary to install with brew

**Comment by Veershah26** [comment]
Thank you for your contribution @sabderemane @luminousbeam . Im merging this PR.

**Pull request merged**